app/api/utility.py

```diff
@@ -1,23 +1,28 @@
 """Constants for federation."""
 
 import os
+import re
 
 import httpx
 from fastapi import HTTPException
 
 #  Neurobagel nodes
-NEUROBAGEL_NODES = os.environ.get("NB_NODES", "https://api.neurobagel.org/")
+NEUROBAGEL_NODES = os.environ.get(
+    "LOCAL_NB_NODES", "(https://api.neurobagel.org/, OpenNeuro)"
+)
 
 
-def parse_nodes_as_list(nodes: str) -> list:
-    """Returns user-defined Neurobagel nodes as a list.
-    Empty strings are filtered out, because they are falsy.
+def parse_nodes_as_dict(nodes: str) -> list:
+    """Returns user-defined Neurobagel nodes as a dict.
+    It uses a regular expression to match the url, name pairs.
     Makes sure node URLs end with a slash."""
-    nodes_list = nodes.split(" ")
-    for i in range(len(nodes_list)):
-        if nodes_list[i] and not nodes_list[i].endswith("/"):
-            nodes_list[i] += "/"
-    return list(filter(None, nodes_list))
+    pattern = re.compile(r"\((https?://[^\s]+), ([^\)]+)\)")
```

Comment:
I think you can simplify this with named capture groups
```suggestion
    pattern = re.compile(r"(?P<url>https?://[^\s]+), (?P<label>[^\)]+)")
```

And then

```python
url = match.group('url')
label = match.group('label')
```